source/Octopus.Server.Client/Operations/RegisterMachineOperation.cs

```diff
@@ -256,13 +338,16 @@ async Task<MachineResource> GetMachine(IOctopusSpaceAsyncRepository repository)
             {
                 existing = await repository.Machines.FindByName(MachineName).ConfigureAwait(false);
             }
-            catch (OctopusDeserializationException) // eat it, probably caused by resource incompatability between versions
+            catch
+                (OctopusDeserializationException) // eat it, probably caused by resource incompatability between versions
```

Comment:
Not you, but I dislike this comment greatly. Can we move it into the catch block and remove the "eat it"?

Reply:
Fixed in fixed in [`073f520` (#845)](https://github.com/OctopusDeploy/OctopusClients/pull/845/commits/073f52085ff8429a3cc073f59d3148ed10ba9589)